nbconvert/exporters/templateexporter.py

```diff
@@ -50,6 +50,7 @@
         'prevent_list_blocks': filters.prevent_list_blocks,
         'get_metadata': filters.get_metadata,
         'convert_pandoc': filters.convert_pandoc,
+        'wrapped_convert_link': wrapped_convert_link,
```

Comment:
I'd like the name here to be a bit more descriptive: `wrapped` is an implementation detail, and it's not clear what it means (wrapped in what?). A descriptive name might be like `pandoc_internal_links_to_latex_refs`. That is a bit wordy, so better suggestions welcome.

I also wonder if it should only be registered for the latex exporter - then the name could be a bit less precise. Adding a filter for an exporter works like this (example from bookbook):

```python
def default_filters(self):
    yield from super(MyHTMLExporter, self).default_filters()
    yield ('markdown2html', markdown2html_custom)
```